web/.env

```diff
@@ -0,0 +1,2 @@
+VUE_APP_I18N_LOCALE=en
+VUE_APP_I18N_FALLBACK_LOCALE=en
```

Comment:
`.env` was added to `.gitignore` a while ago, should this be in the sample file instead (`.env.sample`)?